app/components/community_activity_component/community_activity_component.html.erb

```diff
@@ -82,17 +82,17 @@
                   <div class="icon-close" data-action="click->modal#maybeToggle"></div>
                 <% end %>
                 <% component.with_body do %>
-                    <h3 class="govuk-heading-s"><%= @activity.title %></h3>
+                    <h3 class="govuk-heading-s"><%= @activity.title.html_safe %></h3>
```

Comment:
We should swap this one to `sanitize` as well so that its consistent across the page